examples/pos-mainchain/src/app/modules/testNft/types.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import {
+	LENGTH_COLLECTION_ID,
+	LENGTH_NFT_ID,
+	MAX_LENGTH_MODULE_NAME,
+	MIN_LENGTH_MODULE_NAME,
+} from './constants';
+
+export interface NFTAttributes {
+	module: string;
+	attributes: Buffer;
+}
+
+export const mintNftParamsSchema = {
```

Comment:
minor point but probably this should go inside schema.ts file, also the one below